### PR TITLE
Support @myself in send_update/3

### PR DESCRIPTION
**Pull request by dvic**

### CODE REVIEW

**Comment by dvic** [comment]
@josevalim went ahead and did this PR first, is this what you had in mind?

File: lib/phoenix_live_view.ex [hunk 1]
```diff
@@ -1338,6 +1347,12 @@ defmodule Phoenix.LiveView do
     Phoenix.LiveView.Channel.send_update(pid, module, id, assigns)
```
Comment (josevalim) [inline lib/phoenix_live_view.ex:1347]: Suggestion: make this line be:

```suggestion
    Phoenix.LiveView.Channel.send_update(pid, {module, id}, assigns)
```

This way you can share more code between both branches and disambiguate only in the internals. :)

**Review by josevalim — COMMENTED** [review]
Looks great!! I dropped only comment to help clean the code :)

**Comment by dvic** [comment]
> Looks great!! I dropped only comment to help clean the code :)

Indeed, thanks :) (diff https://github.com/phoenixframework/phoenix_live_view/compare/0bdb046f1fb4f48992a7be2133cb8ab646d946da..5afb2d91d3e22dcb3a281c79feaa6c4df06439b0)

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

File: lib/phoenix_live_view.ex [hunk 2]
```diff
@@ -1327,15 +1330,27 @@ defmodule Phoenix.LiveView do
 
         {:noreply, socket}
       end
+
+      def render(assigns) do
```
Comment (aiwaiwa) [inline lib/phoenix_live_view.ex:1334]: `~H` is missing?

Comment (dvic) [reply lib/phoenix_live_view.ex:1334]: Good catch! I'll fix this in my next pr if it's not fixed by then

**Pull request merged**